core/feconf.py

```diff
@@ -953,6 +953,10 @@ def get_empty_ratings() -> Dict[str, int]:
 FEEDBACK_THREADLIST_URL_PREFIX = '/threadlisthandler'
 FEEDBACK_THREADLIST_URL_PREFIX_FOR_TOPICS = '/threadlisthandlerfortopic'
 FEEDBACK_THREAD_VIEW_EVENT_URL = '/feedbackhandler/thread_view_event'
+FEEDBACK_UPDATES_URL = '/feedback-updates'
```

Comment:
For consistency with the string below, should we rename this to '/feedbackupdateshandler'?

Reply:
Removed as unused